engineering_tools/carma

```diff
@@ -94,10 +123,16 @@ carma-config__inspect() {
 
     echo "Opening shell inside carma-config container with read-only privileges..."
 
-    local carma_base=$(__get_most_recent_carma_base)
+    if [ "$1" = "-d" ]; then
+        local OPTION="-d"
+    elif [ "$1" = "-c" ]; then
+        local OPTION="-c"
+    fi
+
+    local carma_base=$(__get_most_recent_carma_base $OPTION)
     if [[ -z $carma_base ]]; then
-        __pull_newest_carma_base
-        carma_base=$(__get_most_recent_carma_base)
+        __pull_newest_carma_base $OPTION
```

Comment:
I'm uncertain on this. This and the edit command aren't really related to the precise version/config of CARMA-base, since they're just for opening a quick bash shell to maybe edit one of the CARMA configuration files. Not sure how much it matters what CARMA base is being used, as long as it has ROS and other basic tools installed.

Reply:
Oh okay got it. Yeah i'll remove the option then.